installer/cmd/render.go

```diff
@@ -296,3 +285,62 @@ func init() {
 		os.Exit(1)
 	}
 }
+
+// runImports will import all manifests declared in the import interface using parallelism
+func runImports(kImports []importer.KustomizeImporter, yImports []importer.YAMLImporter) (imports []string, err error) {
+	errc := make(chan error)
+	defer close(errc)
+	var errWg sync.WaitGroup
```

Comment:
The x sync library has better groups for this: https://pkg.go.dev/golang.org/x/sync/errgroup

We should be able to make these loops with this `errgroup` package instead, waiting on the whole group. Hopefully, this should make the logic a bit simpler.

Reply:
Interesting, I wasn't aware of this package! Let me try :)